api/v1alpha1/jobset_webhook.go

```diff
@@ -34,43 +34,46 @@ func (r *JobSet) SetupWebhookWithManager(mgr ctrl.Manager) error {
 var _ webhook.Defaulter = &JobSet{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type
-func (r *JobSet) Default() {
-	for i, _ := range r.Spec.ReplicatedJobs {
+func (js *JobSet) Default() {
+	// Default success policy to operator "All" targeting all replicatedJobs.
+	if js.Spec.SuccessPolicy == nil {
+		js.Spec.SuccessPolicy = &SuccessPolicy{Operator: OperatorAll}
```

Comment:
Note for reviewers: nil slice will be functionally the same as an empty slice for our purposes here (`len()` returns 0, can `range` over it (0 iterations) without panic, etc.) so I saw no reason to default to an empty slice.